src/pages/_index.mdx

```diff
@@ -116,7 +116,10 @@ You'll find no subsets like Starlark here. Pants empowers you with full support
 
 Pants supports Python, Docker, Go, Java, Kotlin, Pex, Protodoc, Scala, Shell, Thrift, Protobuf,
 Docker, Helm, many linting and formatting tools, packaging, coverage, and more.
-[Learn more.](/2.19/docs/introduction/welcome-to-pants)
+
+<VersionedDocsLink version="last-stable" unversionedPath="docs/broken-link">
+  Learn more.
+</VersionedDocsLink>
```

Comment:
Note to self: this spacing change is forced by prettier but is changing the display.

Reply:
https://github.com/prettier/prettier/issues/12209